frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -142,7 +152,48 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
           >
             <InlineTruncatedClipboardCopy testId="model-version-id" textToCopy={mv.id} />
           </DashboardDescriptionListGroup>
+          {isPipelineRunExist(mv.customProperties, pipelineRunSpecificKeys) && (
+            <DashboardDescriptionListGroup title="Registered from">
+              <Flex
+                spaceItems={{ default: 'spaceItemsXs' }}
+                alignItems={{ default: 'alignItemsCenter' }}
+                data-testid="registered-from"
+              >
+                <FlexItem data-testid="pipeline-run-link">
+                  Run{' '}
+                  {
+                    <Link
+                      style={{ fontWeight: 'var(--pf-t--global--font--weight--body--bold)' }}
+                      to={globalPipelineRunDetailsRoute(
+                        filteredProperties[pipelineRunSpecificKeys[0]].string_value,
+                        filteredProperties[pipelineRunSpecificKeys[1]].string_value,
```

Comment:
I don't think we should reference these by numeric index, there's probably a safer / more readable way to do this. Like @gitdallas said in his review, maybe we can make a `getRegisteredFromPipelineProperties` function (or however you want to name it) that takes all the customProperties, picks these out as plain strings and returns a typed object so we could reference for example `registeredFromPipelineProperties.project`.